mlabs/src/Mlabs/EfficientNFT/Token.hs

```diff
@@ -90,13 +91,21 @@ mkPolicy burnHash previousNft collectionNftP mintAct ctx =
 
     -- Check if the old token is burnt and new is minted with correct name
     checkMintAndBurn nft newPrice newOwner =
-      let oldName = mkTokenName nft
+      let burntMinted =
+            sortOn (\(_, _, amt) -> amt)
```

Comment:
as in the other PR, not sure if that's the most efficient way of doing this

Reply:
Simplified this